sw/vendor/riscv_compliance/riscv-target/opentitan/compliance_test.h

```diff
@@ -0,0 +1,37 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// RISC-V Compliance Test Header File
+
+#ifndef _COMPLIANCE_TEST_H
+#define _COMPLIANCE_TEST_H
+
+#include "sw/vendor/riscv_compliance/riscv-test-env/p/riscv_test.h"
+
+//-----------------------------------------------------------------------
+// RV Compliance Macros
+//-----------------------------------------------------------------------
+#define RV_COMPLIANCE_HALT                                                    \
+        la sp, _stack_start;                                                  \
+        j dump_signature;                                                     \

```

Comment:
You don't need semicolons here.

Reply:
yeah @gkelly pointed this out, i removed them, got myself in some kind of git hell, and here they are again...

Reply:
actually seems like when i take out the semicolon the build fails, since this is all on one line due to the marcro define.